cookiecutter-django-ida/openedx.yaml

```diff
@@ -0,0 +1,13 @@
+# This file describes this Open edX repo, as described in OEP-2:
```

Comment:
This will be the openedx.yaml file in the created repo, yes?  If so, it should have the "openedx maybe" content from https://github.com/edx/cookiecutter-django-ida/pull/65 ?

Reply:
No, for this file. I'm deleting this cause it will be superseeded by edx-cookiecutter/openedx.yaml file. I'll add the openedx maybe to that file.